pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1455,3 +1457,37 @@ func (l *LibvirtDomainManager) GetGuestInfo() (v1.VirtualMachineInstanceGuestAge
 
 	return guestInfo, nil
 }
+
+// GetUsers return the full list of users on the guest machine
+func (l *LibvirtDomainManager) GetUsers() ([]v1.VirtualMachineInstanceGuestOSUser, error) {
+	userInfo := l.agentData.GetUsers(-1)
+	userList := []v1.VirtualMachineInstanceGuestOSUser{}
+
+	for _, user := range userInfo {
+		userList = append(userList, v1.VirtualMachineInstanceGuestOSUser{
+			UserName:  user.Name,
+			Domain:    user.Domain,
+			LoginTime: user.LoginTime,
+		})
+	}
+
+	return userList, nil
+}
+
+// GetFilesystems return the full list of filesystems on the guest machine
+func (l *LibvirtDomainManager) GetFilesystems() ([]v1.VirtualMachineInstanceFileSystem, error) {
```

Comment:
Error is never returned here?

Reply:
It was a design choice. There is no error to be returned. 
However, having the error returned, the usage is exactly the same as for the other functions, which makes reading the code easier.

I can remove it if this is something that have to be changed.